nebula_ros/launch/continental_launch_all_hw.xml

```diff
@@ -0,0 +1,174 @@
+        </node>
+    </group>
+
+
+    <group if="$(eval &quot;'$(var sensor_model)' == 'SRR520' &quot;)" >
+
+        <node pkg="nebula_ros" exec="continental_srr520_driver_ros_wrapper_node" name="continental_srr520_decoder_driver" output="screen">
+            <param name="sensor_model" value="$(var sensor_model)"/>
+            <param name="frame_id" value="$(var frame_id)"/>
+            <param name="base_frame" value="$(var base_frame)"/>
+
+            <param name="interface" value="$(var interface)"/>
+            <param name="receiver_timeout_sec" value="$(var receiver_timeout_sec)"/>
+            <param name="sender_timeout_sec" value="$(var sender_timeout_sec)"/>
+            <param name="filters" value="$(var filters)"/>
+            <param name="use_bus_time" value="$(var use_bus_time)"/>
+
+            <param name="new_longitudinal_cog" value="1.25"/> <!-- test value -->
+            <param name="new_sensor_id" value="$(var new_sensor_id)"/>
+            <param name="new_vehicle_wheelbase" value="$(var new_vehicle_wheelbase)"/>
+            <param name="new_plug_bottom" value="$(var new_plug_bottom)"/>
+            <param name="new_cover_damping" value="$(var new_cover_damping)"/>
+            <param name="reset_sensor_configuration" value="$(var reset_sensor_configuration)"/>
+        </node>
+
+        <node pkg="nebula_ros" exec="continental_srr520_hw_interface_ros_wrapper_node"
+            name="continental_srr520_hw_driver" output="screen" if="$(var launch_hw)">
+            <remap from="/diagnostics" to="diagnostics"/>
+            <remap from="odometry_input" to="$(var odometry_topic)"/>
+            <remap from="acceleration_input" to="$(var acceleration_topic)"/>
+            <remap from="steering_angle_input" to="$(var steering_angle_topic)"/>
+
+            <param name="sensor_model" value="$(var sensor_model)"/>
+            <param name="frame_id" value="$(var frame_id)"/>
+            <param name="base_frame" value="$(var base_frame)"/>
+
+            <param name="interface" value="$(var interface)"/>
+            <param name="receiver_timeout_sec" value="$(var receiver_timeout_sec)"/>
+            <param name="sender_timeout_sec" value="$(var sender_timeout_sec)"/>
+            <param name="filters" value="$(var filters)"/>
+            <param name="use_bus_time" value="$(var use_bus_time)"/>
+
+            <param name="new_longitudinal_cog" value="1.25"/> <!-- test value -->
+            <param name="new_sensor_id" value="$(var new_sensor_id)"/>
+            <param name="new_vehicle_wheelbase" value="$(var new_vehicle_wheelbase)"/>
+            <param name="new_plug_bottom" value="$(var new_plug_bottom)"/>
+            <param name="new_cover_damping" value="$(var new_cover_damping)"/>
+            <param name="reset_sensor_configuration" value="$(var reset_sensor_configuration)"/>
+
+            <param name="use_sim_time" value="false"/>
+        </node>
+    </group>
```

Comment:
Please move this to #124 if possible :pray:

Reply:
Addressed in [1007571](https://github.com/tier4/nebula/pull/123/commits/100757164b9eee0ced387ed2c5276687796abbfd)